### PR TITLE
add between adapter

### PR DESCRIPTION
**Pull request by Shimogatsu**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/betweenBidAdapter.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Overview
+
+Module Name: Example Bidder Adapter
```
Comment (bretg) [inline modules/betweenBidAdapter.md]: Please update the template and add actual contact info.

File: modules/betweenBidAdapter.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Test Parameters
+```
+    var adUnits = [
+        {
+            code: 'test-div',
+            bids: [
+                {
+                    bidder: "between",
+                    params: {
+                        placementId: '12345',
```
Comment (bretg) [inline modules/betweenBidAdapter.md]: is 12345 an actual bid parameter?

**Comment by bretg** [comment]
Please provide a PR for documentation in https://github.com/prebid/prebid.github.io

File: modules/betweenBidAdapter.md (hunk 2 above)
Comment (Shimogatsu) [reply modules/betweenBidAdapter.md]: I updated pull req and i did pull req in prebid.io

File: modules/betweenBidAdapter.md (hunk 1 above)
Comment (Shimogatsu) [reply modules/betweenBidAdapter.md]: done

**Comment by Shimogatsu** [comment]
i updated pull request

**Comment by bretg** [comment]
my review just covers the docs -- @idettman - please cover the body of the adapter. Thanks.

**Comment by Shimogatsu** [comment]
Guys, do you need anything from me to proceed? What’s our next step here?

**Comment by madm1keru** [comment]
@bretg @idettman please, give us any feedback, we have enough publishers waiting for this pull request to be merged. LEt us know if you need any additional info or file changes.

**Comment by Shimogatsu** [comment]
Done

**Comment by madm1keru** [comment]
@idettman we've fixed merge conflicts without any changes in adapter itself.
Could you pls proceed with this merge?

**Pull request merged**